lang/python/python-pytest/Makefile

```diff
@@ -41,6 +41,8 @@ define Package/python3-pytest
 	+python3-wcwidth \
 	+python3-decimal \
 	+python3-logging \
+	+python3-toml \
+	+python3-iniconfig \
```

Comment:
wcwidth has been [removed](https://github.com/pytest-dev/pytest/blob/6.0.1/doc/en/changelog.rst#trivialinternal-changes-1) as a dependency.

Also, in addition to decimal, logging and urllib, [codecs](https://github.com/pytest-dev/pytest/blob/6.0.1/src/_pytest/_io/wcwidth.py#L1), [pydoc](https://github.com/pytest-dev/pytest/blob/6.0.1/src/_pytest/doctest.py#L171) and [unittest](https://github.com/pytest-dev/pytest/blob/6.0.1/src/_pytest/unittest.py#L68) are needed. I think at this point would be better to replace python3-light and the other stdlib packages with python3.

Reply:
Replace python-light, python3-urllib, python3-wcwidth, python3-decimal, python3-logging with python3 and add dependencies to python-py